hospital/apphospital/models.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
```

Comment:
Read about DB index and add index to relevant fields

Reply:
d